src/Pos/Block/Arbitrary.hs

```diff
@@ -220,8 +219,7 @@ newtype BlockHeaderList ssc = BHL
     } deriving (Eq)
 
 instance T.BiSsc ssc => Show (BlockHeaderList ssc) where
-    show =
-        concat . intersperse "\n" . map (formatToString build) . uncurry zip . getHeaderList
+    show = intercalate "\n" . map (formatToString build) . uncurry zip . getHeaderList
```

Comment:
@gromakovsky I thought `pretty` was you not wanting to write `formatToString build` and abbreviating with `pretty`, as in, the function that makes the headers `pretty`.

Guess next time I should ask first, assume later.